style.css

```diff
@@ -0,0 +1,93 @@
+body{
+  display: flex;
+  flex-direction: column;
+  margin:0;
+  font-family: Arial, Helvetica, sans-serif ;
+}
+.container-home {
+  display: flex;
+  flex-direction: column;
+  margin:0;
+  background-image: url(/images/carousel_img_1.png);
+  background-repeat: no-repeat;
+  height: 600px;
+  background-size: contain;
+  padding-left:30px;
+  
+}
+.container-home h4 {
+  color: white;
+  margin: 0;
+  margin-top:290px;
+  font-size: 35px;
+}
+.container-home p {
+  color: #986627;
+  margin: 0;
+  font-size: 40px;
+  margin-bottom: 30px;
+}
+.button{
+  background-color: transparent;
+  border-color: white;
+  font-size: 50px;
+  width: 60px;
+  margin:5px;
+  border-style: solid;
+}
+.arrow{
+  color: white;
+}
+.container-image {
+  position: relative;
+  height: 350px;
+  padding: 10px;
+}
+.backgroud-image{
+  position:absolute;
+  right: 10px;
+}
+.top-image{
+  position: absolute;
+  top: 135px;
+  left:10px;
+  
+}
+.container-about {
+  padding: 0px 5px 0px 5px;
```

Comment:
You can shorten it like this `padding: 0px 5px`; 
[](https://developer.mozilla.org/en-US/docs/Web/CSS/padding) - here you can read how you can shorten your value and what it means in syntax part. The same goes for margins

Reply:
Also don't need px after 0